### ISSUE TITLE
Incorrect colors

### ISSUE THREAD

**Issue by xSlysoft** [KiranWells/micro-nord-tc-colors#4]
I'm certain that I must be doing something wrong, but all the built in micro color schemes work, so maybe you can help me out. Most of the colors work fine, however others, particularly the background, are clearly wrong (at least, they don't match your screenshots or the nord palette). I am using micro in windows terminal, which does support true color, and I have set MICRO_TRUECOLOR=1, but I am working on a remote server via ssh. Perhaps that is the problem? Strangely, the light version has the white background color, so I'm not sure why the dark is grey instead of blue.

![image](https://user-images.githubusercontent.com/42144975/132615429-a1a4ffef-d031-4c15-9b65-908eafc24db9.png)

**Comment by KiranWells** [comment]
Have you tried changing the $TERM value? Often it acts odd through ssh. Try `export TERM=screen-256color` after logging into the ssh session. 

You can also try `echo $TERM` to see if it is currently set to a recognized 256-color terminal. It is likely that windows terminal is not recognized on that machine.

**Comment by xSlysoft** [comment]
Yeah, echo $TERM returns xterm-256color. I had tried changing it to xterm-24bits in .bash_profile because I had seen that suggested somewhere, but it didn't do anything. Though, I just tried running micro on a local file and it ended up looking even worse than the above screenshot, so there might just be something wrong with my settings.

**Comment by WingofaGriffin** [comment]
Can confirm these colors look off even with screen-256color and xterm-256color set. Looks like the nord maintainer is making an official color scheme now though? https://github.com/arcticicestudio/nord/issues/115
